fs-base/src/main/java/com/palantir/giraffe/file/MoreFiles.java

```diff
@@ -493,71 +492,102 @@ public static boolean isEmpty(Path path) throws IOException {
     }
 
     /**
-     * Given a {@code Path} representing a directory, retrieve the list of files
-     * and directories contained in that {@code Path} as a {@code List}. The
-     * {@code Path} objects are obtained as if by resolving the name of the
-     * directory objects against the given {@code Path}.
+     * Gets a list of the files and directories in the given directory. The
+     * returned paths are {@linkplain Path#resolve(Path) resolved} against the
```

Comment:
I agree that this should be a link/linkplain, but I wanted to get your 2 cents on when to use which.